pixeltable/tests/test_client.py

```diff
@@ -19,3 +97,70 @@ def test_drop_table(self, test_tbl: pt.Table) -> None:
         with pytest.raises(exc.Error):
             _ = t.show(1)
 
+    def test_create_table_from_parquet(self, test_client : pt.Client, tmp_path : Path) -> None:
```

Comment:
those tests don't really have anything to do with the core Client functionality

Reply:
Any suggestions on where to put them?